src/public/admin/earnings/styles.css

```
@@ -60,8 +61,361 @@
+  font-size: 12px !important;
+  font-weight: 600 !important;
+  height: 36px !important;
+  line-height: 36px !important;
+  max-width: 36px !important;
+  transition:
+    background 130ms,
+    border-color 130ms,
+    color 130ms !important;
+}
+
+.flatpickr-day:hover:not(.flatpickr-disabled):not(.selected) {
```

Stylelint (17.5.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)